internal/schema-registry/command_kek_undelete.go

```diff
@@ -0,0 +1,49 @@
+package schemaregistry
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/config"
+	"github.com/confluentinc/cli/v3/pkg/output"
+	"github.com/confluentinc/cli/v3/pkg/resource"
+)
+
+func (c *command) newKekUndeleteCommand(cfg *config.Config) *cobra.Command {
```

Comment:
Should we support multiple un-deletions since we support multiple deletions?

Reply:
sounds good

Reply:
added!